pkg/arm/data/master-startup.sh

```diff
@@ -32,6 +32,7 @@ mkdir -p $DPATH
 base64 -d <<< {{ .Config.Images.ImagePullSecret | Base64Encode }} >${DPATH}/config.json
 chmod 0600 ${DPATH}/config.json
 
+# The additional data disk holds the etcd data in master nodes
```

Comment:
What are we doing with docker on master nodes? it needs some space too. Not that much as nodes, but still

Reply:
@mjudeikis master nodes keep the current configuration where the docker volumes are stored in the temporary drive, which is dependant on the size of the instance

Reply:
for example, in our current QA configuration the master nodes have 16G assigned to `/var/lib/docker`